src/types/submission.ts

```diff
@@ -11,15 +11,20 @@ export enum SubmissionType {
 export interface ISubmission {
   form: IFormSchema['_id']
   authType: AuthType
-  myInfoFields: MyInfoAttribute
+  myInfoFields: MyInfoAttribute[] | []
```

Comment:
just `MyInfoAttribute[]` should be enough, an empty array still fulfils the typing

Reply:
will do